src/app/utils/api/neoscan.js

```diff
@@ -218,7 +243,39 @@ export const get_address_abstracts = (address, page) => {
     })
   })
 }
-//
+
+// Returns the full URL all the way up to the args.
+// I.e., 'https://neoscan.io/api/main_net/v1/get_last_transactions_by_address/'
+// TODO add page argument format = address + '/' + page
+
+exports.get_last_transactions_by_address_url = (address, page) => {
+  if (address) return validateUrl(curState.config.neoscan.active.txsByAddressUrl + '/' + address + '/' + page)
+  else return validateUrl(curState.config.neoscan.active.txsByAddressUrl)
+}
+
+// Get all transactions for an address
+
+exports.get_last_transactions_by_address = (address, page) => {
+  let pageArg = '1'
+  if (page) pageArg = page
```

Comment:
you could rewrite these 2 lines to `let pageArg = page || 1` which defaults to 1 when pageArg is empty/falsely